### DISCUSSION TITLE
能把本地单页应用打包吗？

### DISCUSSION THREAD

**Discussion by tangchi695** [tw93/Pake#579] [Q&A]
我试着用pake打包我用vite+vue3开发的网页
pake .\dist\index.html --name test --iter-copy-file
这样打包以后会报个错，大致是找不到什么  about_pake.html之类的。
然后 pake命令都不能用了。
`
PS C:\Users\LS> pake
node:internal/errors:478
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module 'C:\Users\LS\AppData\Roaming\npm\node_modules\pake-cli\dist\cli.js' imported from C:\Users\LS\AppData\Roaming\npm\node_modules\pake-cli\cli.js
    at new NodeError (node:internal/errors:387:5)
    at finalizeResolution (node:internal/modules/esm/resolve:330:11)
    at moduleResolve (node:internal/modules/esm/resolve:907:10)
    at defaultResolve (node:internal/modules/esm/resolve:1115:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:841:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:76:40)
    at link (node:internal/modules/esm/module_job:75:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}`
只能重新安装cli解决。

**Comment by Tlntin** [comment]
`pake .\dist\index.html --name test --iter-copy-file`
改成

```bash
pake .\dist --name test --iter-copy-file
```
试试。
如果还不行，那我就今晚看看是哪里的问题（可能是年久失修了）

**Reply by tangchi695** [reply]
PS D:\Code\ZJBH> pake .\dist --name ZJBH --iter-copy-file
✼ No icon given, default in use. For a custom icon, use --icon option.
✼ The first use requires installing system dependencies.
✼ See more in https://tauri.app/v1/guides/getting-started/prerequisites.
✺ Installing package...

added 64 packages in 2m

68 packages are looking for funding
✔ Package installed!
✼ The first packaging may be slow, please be patient and wait, it will be faster afterwards.
✼ Your input might be a local file.
node:internal/process/promises:279
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Error: ENOENT: no such file or directory, lstat 'C:\Users\LS\AppData\Roaming\npm\node_modules\pake-cli\dist_bak\about_pake.html'] {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'lstat',
  path: 'C:\\Users\\LS\\AppData\\Roaming\\npm\\node_modules\\pake-cli\\dist_bak\\about_pake.html'
}


会报这个错
而且报这个错以后再次执行 pake命令就会直接报我前面说的错误。

**Reply by Tlntin** [reply]
看起来是about pake这个html被删除了，可能是某次更新误删了，我今晚看看。

**Comment by Tlntin** [comment]
已修复，可以参考该[wiki: Pake打包静态文件示例](https://github.com/tw93/Pake/wiki/Pake%E6%89%93%E5%8C%85%E9%9D%99%E6%80%81%E6%96%87%E4%BB%B6%E7%A4%BA%E4%BE%8B)进行打包。
